client/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@azure/digital-twins-core": "^1.0.3",
     "@azure/identity": "^1.2.4",
+    "@fluentui/react": "^8.56.0",
```

Comment:
I don't think we should add this package.

We are using "office-ui-fabric-react" as the UI framework. This is the *old* name for Fluent UI. Let's see if we can find something in this package rather than adding the newer version side-by-side.

Reply:
Done